map/styled_point.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "map/user_mark.hpp"
+#include "map/user_mark_container.hpp"
+
+#include "search/result.hpp"
+
+#include "indexer/feature.hpp"
+
+#include "geometry/point2d.hpp"
+
+#include "std/string.hpp"
+#include "std/unique_ptr.hpp"
+#include "std/utility.hpp"
+
+namespace style
+{
+// Fixes icons which are not supported by MapsWithMe.
+string GetSupportedStyle(string const & s, string const & context, string const & fallback);
+// Default icon.
+string GetDefaultStyle();
+}  // namespace style
+
+class StyledPoint : public ICustomDrawable
```

Comment:
I see why you're suggesting this, but won't it make it complicated to use? @deathbaba, what do you think?